__tests__/app.test.js

```diff
@@ -180,7 +180,122 @@ describe('POST /api/articles/:article_id/comments', ()=>{
             expect(body.msg).toBe("path not found")
         })
     })
-})
+
+    })
+
+describe('PATCH /api/articles/:article_id', ()=>{
+    test('200 : responds with an updated object with incremented votes given by user at the article_id', ()=>{
+        const newVotes = 10
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes:newVotes})
+        .expect(200)
+        .then(({body})=>{
+            const {article} = body
+            expect(article.votes).toBe(110)
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                article_img_url: expect.any(String),
+                votes: expect.any(Number)
+            })
+        })
+    })
```

Comment:
You can roll these two expect statements into one - instead of expect.any(Number) for votes, you can put 110. This means you wont need your separate expect on line 195